api-reference/v1.0/resources/groups_overview.md

```diff
@@ -0,0 +1,273 @@
+# Working with groups in Microsoft Graph
+
+Groups are collections of [users](*) and [devices](*) (and more?) who share access to resources in Office365 or in your app. All operations in Microsoft Graph on groups require administrator consent. Using Microsoft Graph, you can perform the following operations and more:
+- create groups
+- check if a user is a member of a group
+- list all groups a user is a member of
+- list the members of a group
+- add and remove members from a group
+- add and remove owners of a group.
+
+**NOTE**
+Groups are only supported for enterprise accounts, and not supported for Microsoft personal accounts.
+
+## Office 365 groups
+Office 365 groups are for people who collaborate on a project or a team. Members share access to Office365 resources including:

```

Comment:
You should take a look at this doc:  https://support.office.com/en-us/article/Learn-about-Office-365-groups-b565caa1-5c40-40ef-9915-60fdb2d97fa2 and pull some info from it, and also link to it.